src/Sarif.Viewer.VisualStudio.UnitTests/CodeAnalysisResultManagerTests.cs

```diff
@@ -276,6 +276,39 @@ public void CodeAnalysisResultManager_GetRebaselinedFileName_RelativeUri_Without
             remappedPathPrefixes[0].Item2.Should().Be(@"D:\Users\John\source\sarif-sdk");
         }
 
+        [Fact]
+        public void CodeAnalysisResultManager_GetRebaselinedFileName_ConstructPathFromSolutionPathRemapping()
```

Comment:
>CodeAnalysisResultManager_GetRebaselinedFileName_ConstructPathFromSolutionPathRemapping [](http://example.com/codeflow?start=20&length=87)

are you testing two functionalities at the same time? If yes, can you split the tests?

Reply:
Can path be null? if yes, are we covering that? If no, we don't need the path?....in the normalize function that you created

Reply:
its a like a sequence steps, the issue happens when construct file path using the cached solution path. So the first step need to setup a paths mapping. the second step to resolve the file path.

The path can be empty string, ideally it should not be null, it cannot enter this code. But we should have test to cover it will add test case